CMakeLists.txt

```diff
@@ -34,8 +34,7 @@ OPTION( PROFILE				"Generate profiling information"		OFF )
 OPTION( ENABLE_TRACE		"Enables tracing support"				OFF )
 OPTION( LIBGIT2_FILENAME	"Name of the produced binary"			OFF )
 
-OPTION( ANDROID				"Build for android NDK"	 				OFF )
-
+OPTION( USE_OPENSSL                     "Link with and use openssl library"             ON )
```

Comment:
Where does this line come from? We don't use this at all either in this commit or before.